lang/cs/Org.Apache.REEF.Tests/Functional/Bridge/TestCloseTask.cs

```diff
@@ -69,6 +74,40 @@ public void TestStopTaskOnLocalRuntime()
         }
 
         /// <summary>
+        /// This test is to close a running task and enforce it to break and return after the current iteration
+        /// </summary>
+        [Fact]
+        public void TestBreakTaskOnLocalRuntime()
```

Comment:
REEF-1408 is to test real IMRU task. It is pending for sending PR as it has dependency on this one. As it is not small test case and involves several classes, I would prefer to make it as a separate PR.